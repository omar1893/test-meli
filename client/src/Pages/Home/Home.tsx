import React from 'react'
import './Home.scss'

const Home = () => {
  return (
    <div className='homeContainer'>
      <h2>Bienvenido a Mercado Libre</h2>
      <p>Empieza usando el buscador en la parte de ariba para encontrar todos los productos que quieras.</p>
    </div>
  )
}

export default Home