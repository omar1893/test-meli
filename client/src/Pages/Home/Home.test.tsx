import React from "react";
import { render } from "@testing-library/react";
import Home from "./Home";

describe("Home", () => {
  it("Should match snapshot", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});