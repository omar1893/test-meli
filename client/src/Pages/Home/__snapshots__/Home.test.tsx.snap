// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Should match snapshot 1`] = `
<div>
  <div
    class="homeContainer"
  >
    <h2>
      Bienvenido a Mercado Libre
    </h2>
    <p>
      Empieza usando el buscador en la parte de ariba para encontrar todos los productos que quieras.
    </p>
  </div>
</div>
`;
